packages/ui-components/src/components/ContextMenu/ContextMenu.component.js

```diff
@@ -60,7 +61,7 @@ export const ContextMenu = ({
 
   return (
     <Menu>
-      <Float>
+      <Float middleware={[flip()]}>
```

Comment:
I follow a simple rule:
You touched the functionality of a component(change behaviour, extend or delete).
You add the test :)

Reply:
If you have an idea how to properly wrap the positioning check of the subcomponent into a unit test check, I'm open to implement one. In the PR comment I referred to the previous PR not having a dedicated unit test for its case either.

Reply:
You reference the fix from https://github.com/cloudoperators/juno/pull/204 
However there to fix the issue we removed the flip middleware and replaced it with autoPlacement.  I recommend you do the same here so that they behave the same.

Reply:
I would rather request the opposite. That I change the `autoPlacment` in that component to `flip` as well.
Because I noticed a bug by switching to public (npm) juno.
Autoplacement aligns the subcomponent to the area that has more space left. This means that the `Select` component is aligned downwards if there is more space or upwards, if there is more space left.

You can see it in the following example:
<img width="173" alt="image" src="https://github.com/user-attachments/assets/185c2f4d-309a-41e0-a567-a8c0dbd9e47c">

The contents are aligned upwards, because there is more screenspace left, not because we are out of space.

Reply:
yes, this was described in the referenced PR as well. `flip`, if it works in all cases, would definitely be preferred behaviour but as I mentioned, there were edge cases where it didn't work as expected. autoPlacement is not ideal and I would prefer flip but then we'll have to fix those edge cases. 

We are currently working on a fix to another esoteric placement issue with the DateTimePicker. Maybe we're lucky and the fix there also fixes the placement problems for the flip middleware. We'll have a look.

Reply:
Thank you for you insight, I wasn't fully aware of the edge cases here. I will include the `autoPlacement` config.

Reply:
The config now matches the `Select` counterpart. I had to change the allowed placements to match the desired behavior of the ContextMenu specifically tho.

There is also an issue with the overflow wich is present in the `Select` component - if you scroll down enough not all items are accessible in within the scroll list. This changes the further up you scroll in the documentation. This might be an issue originating from the box Storybooks box resizing.
This is now present in the ContextMenu as well. However if you look at the placement issue anyways, it is as you said, better to have matching behaviors on both components.